### PR TITLE
Poo#18282

### PR DESCRIPTION
**Pull request by mimi1vx**
Use specific_bootmenu_params from bootloader_setup in ofw bootloader.
So all parameters set in in this function are used also in ppc64 tests.

http://furud.arch.suse.de/tests/14#step/bootloader_ofw/10
http://furud.arch.suse.de/tests/16#step/bootloader_ofw/10

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -216,14 +216,8 @@ sub specific_bootmenu_params {
...
-        if ($self_update_repo eq "1") {
-            diag "Explicitly enabling installer self update";
-            $args .= " self_update=1";
-        }
-        else {
-            diag "Explicitly enabling installer self with $self_update_repo";
-            $args .= " self_update=$self_update_repo";
-        }
+        $args .= " self_update=$self_update_repo";
+        diag "Explicitly enabling installer self with $self_update_repo";
```
Comment (dzedro) [inline lib/bootloader_setup.pm]: s/self/self update/

File: tests/installation/bootloader_ofw.pm [hunk 2]
```diff
@@ -67,33 +73,10 @@ sub run() {
...
-                  if defined get_var("NETWORK_INIT_PARAM");    #e.g netsetup=dhcp,all
-                $args .= $netsetup;
-                $args .= " autoyast=" . data_url(get_var("AUTOYAST")) . " ";
-            }
-
-            if (get_var("AUTOUPGRADE")) {
-                $args .= " autoupgrade=1";
-            }
 
             type_string_slow $args;
```
Comment (dzedro) [inline tests/installation/bootloader_ofw.pm]: please change this to type_string_very_slow

File: tests/installation/bootloader_ofw.pm [hunk 3]
```diff
@@ -37,7 +38,12 @@ sub run() {
         || get_var('AUTOYAST')
         || get_var('SCC_URL')
         || get_var('DUD')
-        || get_var('EXTRABOOTPARAMS'))
+        || get_var('EXTRABOOTPARAMS')
+        || get_var('FIPS')
+        || get_var('AUTOUPGRADE')
+        || get_var('INSTALLER_SELF_UPDATE')
+        || get_var('INSTALLER_NO_SELF_UPDATE')
+        || get_var('IFTB'))
```
Comment (dzedro) [inline tests/installation/bootloader_ofw.pm]: what is IFTB ? :)

Comment (dzedro) [inline tests/installation/bootloader_ofw.pm]: rofl http://www.urbandictionary.com/define.php?term=iftb

**Comment by okurz** [comment]
cool, LGTM

**Pull request merged**